### ISSUE TITLE
The Possible Limitation of Crowd Wisdom - Difference between Online and Offline Team Collaboration

### ISSUE THREAD

**Issue by xiuyuanzhang** [uchicago-computation-workshop/james_evans#1]
Thank you for presenting.

This topic on how political beliefs, a heated ideological difference nowadays, among "team members" can influence the quality of work that said team produced is very interesting. I find the usage of lexical and semantic diversity to quantify information diversity on Wikipedia Talk page  a really interesting as well as informative one. (Also I didn't know that there is a Talk page, and it is super interesting to read the discussions going on there.)

My question is about the external validity of this idea on crowd wisdom among offline communities (or teams.)  What seems to me is unique about this paper and the study is that Wikipedia is a gigantic crowd-sourcing "team" if one could even call it that. My concern is that the number of people involved in editing a Wiki page and their loose connection with each other would be a confounding factor. This intimacy factor, for me, becomes a possible obstacle for team members in a lab, company or any other real-life settings to correct each other or disagree with each other. 

To sum up, does this idea of "crowd wisdom" only have positive impact on quality of work when one controls for the setting that crowd-sourcing happening as well as the personal relationship (or the absence of it) among the participants?

**Comment by lpwarner** [comment]
Really interesting point. Do the authors have an idea to what extent Wikipedia editors are aware of each others political biases? E.g. here might Gator1 and Gregmg have any understanding of each other's identity?

![screen shot 2018-01-24 at 5 11 54 pm](https://user-images.githubusercontent.com/31451410/35362141-b7936310-0129-11e8-84ff-a5328a215663.png)
